File: 프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.cs
using System;

public class Solution {
    public string solution(int age) {
        string[] ageStrList = {"a","b","c","d","e","f","g","h","i","j"};
        
        string ageStr = age.ToString();
        string result = "";
        for(int i=0; i<ageStr.Length; i++)
        {
            result += ageStrList[Int32.Parse(ageStr[i].ToString())];
        }
        return result;
    }
}

File: 프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.cpp
#include <string>
#include <vector>
#include <map>

using namespace std;

string solution(int age) 
{
    string result = "";
    map<char, char> ageMap = 
    {
        {'0','a'},{'1','b'},{'2','c'},{'3','d'},{'4','e'},
        {'5','f'},{'6','g'},{'7','h'},{'8','i'},{'9','j'}
    };
    for(char &c : to_string(age))
    {
        result += ageMap[c];
    }
    return result;
}